code/mod_table/mod_table.cpp

```diff
@@ -48,6 +48,24 @@ SCP_string Window_icon_path;
 bool Disable_built_in_translations;
 bool Weapon_shockwaves_respect_huge;
 bool Using_in_game_options;
+ubyte Arc_color_damage_p1_r = 64;
+ubyte Arc_color_damage_p1_g = 64;
+ubyte Arc_color_damage_p1_b = 225;
+ubyte Arc_color_damage_p2_r = 128;
+ubyte Arc_color_damage_p2_g = 128;
+ubyte Arc_color_damage_p2_b = 255;
+ubyte Arc_color_damage_s1_r = 200;
+ubyte Arc_color_damage_s1_g = 200;
+ubyte Arc_color_damage_s1_b = 255;
+ubyte Arc_color_emp_p1_r    = 64;
+ubyte Arc_color_emp_p1_g    = 64;
+ubyte Arc_color_emp_p1_b    = 5;
+ubyte Arc_color_emp_p2_r    = 128;
+ubyte Arc_color_emp_p2_g    = 128;
+ubyte Arc_color_emp_p2_b    = 10;
+ubyte Arc_color_emp_s1_r    = 255;
+ubyte Arc_color_emp_s1_g    = 255;
+ubyte Arc_color_emp_s1_b    = 10;
```

Comment:
The default values should be set in `mod_table_reset` to make the system easier to test.

Reply:
Will do!